src/lib/guesstimator/samplers/DistributionLognormal.js

```diff
@@ -11,7 +10,7 @@ export var Sampler = {
 
     const mean = math.mean(logHigh, logLow)
     const stdev = (logHigh-logLow) / (2*1.645)
-    return { values: Sample(n, () => jStat.lognormal.sample(mean, stdev)) }
+
+    return simulate(`lognormal(${mean},${stdev})`, [], n)
```

Comment:
I think this feels more like a step of the formatter to me, but let's change that later.